### PR TITLE
Update KA Lite OS X installer documentation for 0.17.x release

### PR DESCRIPTION
**Pull request by mrpau-richard**
## Summary

> I updated the README.md and the release notes for 0.17.x release.

/cc @cpauya @mrpau-eduard

### CODE REVIEW

File: osx/KA-Lite/KA-Lite/README.md [hunk 1]
```diff
@@ -6,20 +6,20 @@ This is the Xcode project for the KA Lite application.
 
 ## Requirements
 
-Xcode 6.x or 7.x
+Xcode 7.2
```
Comment (cpauya) [inline osx/KA-Lite/KA-Lite/README.md]: Xcode 7.2 or higher?  Since macOS Sierra I think has Xcode 8.x already and we've tested this works on that too.

File: osx/release-docs/README.md [hunk 2]
```diff
@@ -67,7 +69,7 @@ To view the KA Lite installer and application logs, launch the `Console` applica
 
 To view the KA Lite web server logs, open `~/.kalite/server.log` to view access and debug logs.  If you have set a custom KA Lite data folder in the Preferences dialog, use this format: `$KALITE_HOME/server.log`.
 
-If you encounter issues, please file them at the [KA Lite repository](https://github.com/learningequality/ka-lite/issues/) or at the [KA Lite Installers repository](https://github.com/learningequality/installers).
+If you encounter issues, please file them at the [KA Lite repository](https://github.com/learningequality/ka-lite/issues/) or at the [KA Lite Installers repository](https://github.com/learningequality/ka-lite-installers/issues/).
 
 Please note that we have tested this application on the following Mac OS X versions:
```
Comment (cpauya) [inline osx/release-docs/README.md]: Please add macOS Sierra here.

File: osx/release-docs/RELEASE_NOTES.md [hunk 3]
```diff
@@ -1,6 +1,28 @@
 Release Notes for KA Lite Mac OS X Application
 ==============================================
 
+0.17.0
+------
+
+**Mac Installer**
+
+* NEW - We now support Mac OS Sierra.
```
Comment (cpauya) [inline osx/release-docs/RELEASE_NOTES.md]: I think the proper name is `macOS Sierra`: https://www.apple.com/macos/sierra/

File: osx/release-docs/RELEASE_NOTES.md [hunk 4]
```diff
@@ -1,6 +1,28 @@
...
+* NEW - We now required Python 2.7.11+ installed on your machine to install KA Lite.
```
Comment (cpauya) [inline osx/release-docs/RELEASE_NOTES.md]: Can you rephrase this please?  Try "We now require Python 2.7.12 or higher to install KA Lite."?

**Comment by mrpau-richard** [comment]
@cpauya I updated all the README.md and release notes for 0.17.x.

**Comment by cpauya** [comment]
Can you post one or screenshots of how the `.dmg` window and the `.pkg` window with `README.rst` looks?

**Comment by mrpau-richard** [comment]
Here's the screenshot of .`dmg`
screen shot 2017-01-25 at 4 40 11 pm

File: osx/release-docs/README.md [hunk 5]
```diff
@@ -5,13 +5,15 @@ This is the [KA Lite](https://github.com/learningequality/ka-lite/) application
...
+It uses [PEX](https://pex.readthedocs.io/en/stable/) to create `kalite.pex` executable.
 
 
 ## Install KA Lite
 
-1. Download the [KA Lite Mac OS X Installer for 0.16](http://pantry.learningequality.org/downloads/ka-lite/0.16/installers/mac/).
-1. Double-click the downloaded `KA-Lite.pkg` package and click "Continue" when prompted about the "This package will run a program to determine if the software can be installed." message.
+1. Download the [KA Lite Mac OS X Installer for 0.17](http://pantry.learningequality.org/downloads/ka-lite/0.17/installers/mac/).
+1. Double-click the downloaded `KA-lite-Installer.dmg`.
+1. KA Lite Mac OS X Installer required Python 2.7.12 or higher versions. If you're not sure the version of python installed in your machine, double-click README.md for the instructions.
```
Comment (cpauya) [inline osx/release-docs/README.md]: Please re-phrase to `... If you're not sure which version of Python is installed in your machine...`?

**Comment by cpauya** [comment]
@mrpau-richard Can you re-target `0.17.x` branch for this coz `develop` is now targeting `0.18.x`.  I will resume review on that new PR.

**Comment by benjaoming** [comment]
Sorry for the confusion @cpauya - 0.17.x has not been created in the installers repo yet. We can wait until we start developing 0.18 :)

File: osx/KA-Lite/KA-Lite/README.md [hunk 6]
```diff
@@ -6,20 +6,20 @@ This is the Xcode project for the KA Lite application.
...
 1. Preferences dialog for admin tasks like setting a custom KA Lite data path.
 1. Logs messages so they are accessible at the Logs tab of the Preferences dialog and the Console app.
 
 
 ## Notes:
-
-* The application will not load if the `/usr/local/bin/kalite` executable cannot be found.
+* The application will not load if the `kalite.pex` executable cannot be found.
 * Refer to the `osx/release-docs/README.md` document for more details.
+* The application uses `KALITE_PEX` environment variable to get the path of `kalite.pex` executable and if the environment will not exist, it will default to "/Applications/KA-Lite/support/ka-lite/kalite.pex".
```
Comment (cpauya) [inline osx/KA-Lite/KA-Lite/README.md]: Can you change `executable and if the environment will not exist, it will default` to `executable and if it does not exist, it will default`?

File: osx/release-docs/RELEASE_NOTES.md [hunk 7]
```diff
@@ -1,6 +1,28 @@
...
 
+0.17.0
+------
+
+**Mac Installer**
+
+* NEW - We now support macOS Sierra.
+* NEW - We now require Python version 2.7.12 or higher except Python 3.x.x to install KA Lite.
+* NEW - Add Python 2.7.12 installer to KA-Lite-Installer.dmg.
+* NEW - Add README.md to KA-Lite-installer.dmg.
```
Comment (cpauya) [inline osx/release-docs/RELEASE_NOTES.md]: Should be `KA-Lite-Installer.dmg`?

**Comment by cpauya** [comment]
No worries @benjaoming - pretty minor changes requested and this should be good to merge @mrpau-richard. 😸
